src/framework/preferences/js/Builder.js

```diff
@@ -196,12 +196,22 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     /*
-     * A one-stop-shop function to build and instantiate a prefsEditor from a schema.
+     * A one-stop-shop component to build and instantiate a prefsEditor from a schema.
      */
-    fluid.prefs.create = function (container, options) {
-        options = options || {};
-        var builder = fluid.prefs.builder(options.build);
-        return fluid.invokeGlobalFunction(builder.options.assembledPrefsEditorGrade, [container, options.prefsEditor]);
-    };
+    fluid.defaults("fluid.prefs.create", {
+        gradeNames: ["fluid.viewComponent"],
+        builder: {}, // options provided by users
+        prefsEditor: {}, // options provided by users
+        members: {
+            constructedGrades: "@expand:fluid.prefs.builder({that}.options.build)"
+        },
+        components: {
+            prefsEditor: {
+                type: "{fluid.prefs.create}.constructedGrades.options.assembledPrefsEditorGrade",
```

Comment:
Sorry I am a bit below par today. The approach itself is fine, but we might want to worry about the fact that the returned component from this grade does not do anything useful until the user looks at the "prefsEditor" inside it. Do we have any examples in our documentation of expecting the user to be able to do anything with the component directly?